### PR TITLE
[v0.0.9] Manage simple errors in tesla-control

### PR DESCRIPTION
**Pull request by raphmur**
Capture stdout & stderr to identify errors.

So far only messages starting with `Failed to execute command: car could not execute command` are processed. In this case, the car has received the command, it is valid, but for a reason (not processed) cannot execute it.
Eg. is already charging, charge limit already set, AC already on, ...

Site commit: warn on standalone broken for now

### CODE REVIEW

**Comment by epenet** [comment]
Could the version be added to the MQTT device information?

**Comment by raphmur** [comment]
> Could the version be added to the MQTT device information?

You mean adding the add-on version as an MQTT entity?

**Comment by epenet** [comment]
I think it should be straightforward adding `sw_version` to 
https://github.com/tesla-local-control/tesla-local-control-addon/blob/a44f6b561decc03bdfc269ba5cbf2640b16ca752/tesla_ble_mqtt/rootfs/app/discovery.sh#L8

**Comment by raphmur** [comment]
Ok great thanks.
Can you open an issue? Or even a PR... :)

**Comment by epenet** [comment]
I opened issue #41
Until I can get standalone working I cannot test any of my changes so I don't feel confortable opening PRs.

**Comment by raphmur** [comment]
> I opened issue #41 Until I can get standalone working I cannot test any of my changes so I don't feel confortable opening PRs.

Ok thanks, At least we track your idea. Standalone should be working by the end of the week latest

File: tesla_ble_mqtt/rootfs/app/run.sh [hunk 1]
```diff
@@ -58,19 +58,27 @@ else
...
   else
-    bashio::log.error "tesla-control send command failed exit status $EXIT_STATUS. Retrying in $SEND_CMD_RETRY_DELAY"
+	if [[ $message == *"Failed to execute command: car could not execute command"* ]]; then
+	  bashio::log.error $message
+	  bashio::log.notice "Skipping command $1"
+	  break
+	else
+      bashio::log.error "tesla-control send command failed exit status $EXIT_STATUS."
+	  bashio::log.info $message
+	  bashio::log.notice "Retrying in $SEND_CMD_RETRY_DELAY"
```
Comment (baylanger) [inline tesla_ble_mqtt/rootfs/app/run.sh]: We could add "seconds" in a future PR or now.

Comment (baylanger) [reply tesla_ble_mqtt/rootfs/app/run.sh]: Done.

**Review by baylanger — COMMENTED** [review]
Fantastic work!

**Comment by raphmur** [comment]
we all revied. I am force pushing this one.

**Pull request merged**